api/tests/core/users/test_api.py

```diff
@@ -153,7 +153,7 @@ def test_get_user_with_valid_token_with_expired_date(self):
         expiration_date = datetime.datetime.utcnow() - datetime.timedelta(hours=24)
         users_factories.TokenFactory(user=user, value=self.token_value, type=token_type, expirationDate=expiration_date)
 
-        assert users_models.Token.query.filter_by(value=self.token_value).first() is not None
+        assert not token_utils.SixDigitsToken.token_exists(token_utils.TokenType.PHONE_VALIDATION, user.id)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Même question sur cette assertion :slightly_smiling_face:

Reply:
même réponse

Reply:
c'est fait partout